backend/api/Areas/Project/Controllers/DisposeController.cs

```diff
@@ -116,13 +121,31 @@ public async Task<IActionResult> AddProjectAsync(ProjectModel model)
         [ProducesResponseType(typeof(ProjectModel), 200)]
         [ProducesResponseType(typeof(ErrorResponseModel), 400)]
         [SwaggerOperation(Tags = new[] { "project" })]
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "To support standardized routes (/update/{id})")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "To support standardized routes (/{projectNumber})")]
         public async Task<IActionResult> UpdateProjectAsync(string projectNumber, ProjectModel model)
         {
             var project = _pimsService.Project.Get(model.Id);
             var fromStatusId = project.StatusId;
-            project = _pimsService.Project.Update(_mapper.Map<Entity.Project>(model));
-            var notifications = _pimsService.NotificationQueue.GenerateNotifications(project, fromStatusId, project.StatusId);
+            var updatedProject = _mapper.Map<Entity.Project>(model);
+
+            // Determine if there are any response changes.
+            var responses = project.GetResponseChanges(updatedProject);
+
+            project = _pimsService.Project.Update(updatedProject);
+
+            var notifications = new List<NotificationQueue>();
+            // If the status did not change, do not send notifications.
+            if (fromStatusId != model.StatusId)
+            {
+                notifications.AddRange(_pimsService.NotificationQueue.GenerateNotifications(project, fromStatusId, model.StatusId));
```

Comment:
I contemplated including more of the notification logic in the DAL, but felt it wasn't an ideal solution either.